deploy/007-deploy-xvs.ts

```diff
@@ -0,0 +1,19 @@
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  await deploy("XVS", {
+    from: deployer,
+    args: [deployer],
+    log: true,
+    autoMine: true,
+  });
+};
+
+func.tags = ["xvs"];
+
+export default func;
```

Comment:
We should skip deployment if network is live

Reply:
Rather than skipping it should we modify it to deploy the correct XVS contract based on the network and skip on BSC since it is already deployed there?

Reply:
Makes sense I already added skipping it on a live network